contracts/src/PDPVerifier.sol

```diff
@@ -229,7 +230,15 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
     // A proof set is created empty, with no roots. Creation yields a proof set ID
     // for referring to the proof set later.
     // Sender of create message is proof set owner.
-    function createProofSet(address listenerAddr) public returns (uint256) {
+    function createProofSet(address listenerAddr) public payable returns (uint256) {
+        uint256 sybilFee = PDPFees.sybilFee();
+        require(msg.value >= sybilFee, "sybil fee not met");
```

Comment:
This check is already performed in `burnFee`. It's confusing to have it here but not in `provePosession`.